developer_manual/digging_deeper/continuous_integration.rst

```diff
@@ -43,28 +45,34 @@ You can validate the ``info.xml`` :ref:`app metadata<app metadata>` file of an a
 php
 ^^^
 
-A lint of all php source files can find syntax errors that could crash the application in production. The github action below uses a test matrix of multiple php versions. Adjust it to the ones your app supports.
+A lint of all php source files can find syntax errors that could crash the application in production. You can find the github actions in our `nextcloud template repository <https://github.com/nextcloud/.github>`_.
+You will also require the following lint script in your ``composer.json``:
 
-.. code-block:: yaml
+.. code-block:: json
+
+  {
+    "scripts": {
+      "lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l"
+    }
+  }
+
+php-cs
+^^^^^^
+
+We encourage the usage of php-cs linting. You can find some documentation on how to set this up in the
+`nextcloud coding-standard repository <https://github.com/nextcloud/coding-standard>`_ as well as the
+relevant github actions in our `nextcloud template repository <https://github.com/nextcloud/.github>`_.
+You will also require the following scripts in your ``composer.json``:
+
+.. code-block:: json
+
+  {
+    "scripts": {
+      "cs:fix": "php-cs-fixer fix",
+      "cs:check": "php-cs-fixer fix --dry-run --diff"
+    }
+  }
```

Comment:
Do you also want me to remove this @ChristophWurst ?

Reply:
yes, let's only have this at one place – the readme :+1:

Reply:
Done! :rocket: